From: =?utf-8?q?Tim_R=C3=BChsen?= <tim.ruehsen@gmx.de>
Date: Sun, 27 Sep 2015 19:14:58 +0200
Subject: Extend tools/psl --print-info and --version

---
 tools/psl.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/tools/psl.c b/tools/psl.c
index 6790efe..fb3972c 100644
--- a/tools/psl.c
+++ b/tools/psl.c
@@ -117,7 +117,7 @@ int main(int argc, const char *const *argv)
 				usage(0, stdout);
 			}
 			else if (!strcmp(*arg, "--version")) {
-				printf("psl %s\n", PACKAGE_VERSION);
+				printf("psl %s (0x%06x)\n", PACKAGE_VERSION, psl_check_version_number(0));
 				printf("libpsl %s\n", psl_get_version());
 				printf("\n");
 				printf("Copyright (C) 2014-2015 Tim Ruehsen\n");
@@ -214,6 +214,7 @@ int main(int argc, const char *const *argv)
 			printf("builtin compile time: %ld (%s)\n", psl_builtin_compile_time(), time2str(psl_builtin_compile_time()));
 			printf("builtin file time: %ld (%s)\n", psl_builtin_file_time(), time2str(psl_builtin_file_time()));
 			printf("builtin SHA1 file hash: %s\n", psl_builtin_sha1sum());
+			printf("builtin outdated: %d\n", psl_builtin_outdated());
 		} else
 			printf("No builtin PSL data available\n");
 	}
